src/mailadm/mailcow.py

```diff
@@ -62,7 +63,7 @@ def get_user(self, addr):
     def get_user_list(self):
         """HTTP Request to get all mailcow users (not only mailadm-generated ones)."""
         url = self.mailcow_endpoint + "get/mailbox/all"
-        result = r.get(url, headers=self.auth, timeout=30)
+        result = r.get(url, headers=self.auth, timeout=HTTP_TIMEOUT)
```

Comment:
actually, 5 seconds might be too short here. Right now we have severe problems with dc.develcow.de, the database is under heavy load (some 1.2 million SQL requests in 2 minutes). It's yet unclear where they are from, the servercow people are investigating.

I just tried to do a `get_user_list()` web request and it took 29.94 seconds. I'm not sure how to proceed here, but we should not merge this at this point; pruning and account creation don't rely on this method, but some tests need it and run into HTTP errors already with the busy API.

I'll let you know if we find out the reason for the slow API/busy database.

Reply:
Okay, we are down to 5.something seconds per request. I would have a larger timeout for the "get all accounts" request, as the associated mailcow-internal DB call is more complicated, involves many subqueries and takes more time. For the other requests 5 seconds should be fine now.